src/sys/windows.rs

```diff
@@ -42,6 +42,9 @@ const WSA_FLAG_OVERLAPPED: DWORD = 0x01;
 
 pub use winapi::ctypes::c_int;
 
+/// Fake MSG_TRUNC flag for the [`RecvFlags`] struct.
+pub(crate) const MSG_TRUNC: c_int = sock::WSAEMSGSIZE;
```

Comment:
Wasn't too sure about this: using the error code as "flag". Probably a bad idea now that I think about it. It's highly unlikely that it is a power of two. I'll just copy the value from Linux instead.

Reply:
Replaced with 0x20 and a comment that it's copied from Linux.